ciri/BUILD

```diff
@@ -1,19 +1,10 @@
 package(default_visibility = ["//visibility:public"])
 
-genrule(
-    name = "snapshot_testnet",
-    outs = ["snapshotTestnet.txt"],
-    cmd = "wget -q https://raw.githubusercontent.com/iotaledger/iri/dev/src/main/resources/$$(basename $@) -O $@",
-)
-
 cc_binary(
     name = "ciri",
     srcs = ["main.c"],
     copts = ["-DLOGGER_ENABLE"],
-    data = [
-        "ciri.db",
-        ":snapshot_testnet",
-    ],
+    data = ["ciri.db"],
```

Comment:
Isn't this generated via some genrule elsewher? if so, we should reference the rule & not the file here